bench/CSV.hs

```diff
@@ -109,20 +109,20 @@ module Main (main) where
 
 import qualified Data.ByteString                     as S
 import qualified Data.ByteString.Lazy                as L
+import qualified "bytestring" Data.ByteString.Lazy   as BaseL
```

Comment:
Why is it important to import `BaseL.length` and not `L.length`?

Reply:
`BaseL.length` is applied to output from `text`, which uses the _installed_ version of `bytestring`, not the one from this repo.

The `bench/cabal.project` could be adjusted to include our local `bytestring` package, but that would mean that after each change to the library, the benchmark dependencies would have to be rebuilt.

Reply:
Thanks. It's worth to add a comment about this.

Reply:
> `BaseL.length` is applied to output from `text`, which uses the _installed_ version of `bytestring`, not the one from this repo.

To be more precise: The `Data.ByteString.Lazy` module imported on line 111 is part of `bench-bytestring`. The one on line 112 is from the `bytestring` package.

Reply:
Yes, this is not obvious. A comment explaining would help.

Reply:
Added a comment.